external/vcm/setup.py

```diff
@@ -24,11 +24,12 @@
     "xgcm",
     "cftime",
     "pytest",
-    "google-cloud-storage>=1.28.0",
+    "google-cloud-storage>=1.18.1",
```

Comment:
Do you need to re-make the constraints.txt?

Reply:
No. The new version range contain the old one. so any lock file satisfying the old will also satisfy the new.